end-to-end-untargeted-metabolomics.Rmd

```diff
@@ -1169,73 +1189,131 @@ eic_met <- eic_is["methionine_13C_15N"]
+before and after alignment for both the isotope labeled cystine and methionine.
 
 ```{r specific ion before and after1}
-par(mfrow = c(1, 2), mar = c(4, 4.5, 2, 1))
+par(mfrow = c(2, 2), mar = c(4, 4.5, 2, 1))
 
 old_eic_cystine <- eics_is_refined["cystine_13C_15N"]
 plot(old_eic_cystine, main = "Cystine before alignment", peakType = "none",
      col = paste0(col_sample, 80))
 grid()
 abline(v = intern_standard["cystine_13C_15N", "RT"], col = "red", lty = 3)
 
+old_eic_met <- eics_is_refined["methionine_13C_15N"]
+plot(old_eic_met, main = "Methionine before alignment",
+     peakType = "none", col = paste0(col_sample, 80))
+grid()
+abline(v = intern_standard["methionine_13C_15N", "RT"], col = "red", lty = 3)
+
 plot(eic_cystine, main = "Cystine after alignment", peakType = "none",
      col = paste0(col_sample, 80))
 grid()
 abline(v = intern_standard["cystine_13C_15N", "RT"], col = "red", lty = 3)
 legend("topright", col = col_phenotype,
        legend = names(col_phenotype), lty = 1, bty = "n")
-```
-
-The non_endogenous cystine ion was already well aligned so the difference is
-minimal. The methionine ion, however, shows a clear improvement in alignment:
-
-```{r echo=FALSE}
-par(mfrow = c(1, 2), mar = c(4, 4.5, 2, 1))
-old_eic_met <- eics_is_refined["methionine_13C_15N"]
-plot(old_eic_met, main = "Methionine before alignment",
-     peakType = "none", col = paste0(col_sample, 80))
-grid()
-abline(v = intern_standard["methionine_13C_15N", "RT"], col = "red", lty = 3)
 
 plot(eic_met, main = "Methionine after alignment",
      peakType = "none", col = paste0(col_sample, 80))
 grid()
 abline(v = intern_standard["methionine_13C_15N", "RT"], col = "red", lty = 3)
 ```
 
-NOTE: i would like to maybe not base the alignment on the QC samples. As it is
-actually making the ions test worse.
+The non_endogenous cystine ion was already well aligned so the difference is
+minimal. The methionine ion, however, shows an improvement in alignment.
+
+In addition to a visual inspection of the results, we next also evaluate the
+impact of the alignment by comparing the variance in retention times for
+internal standards before and after alignment. To this end we need to first
+identify chromatographic peaks in each sample with an *m/z* and retention time
+close to the expected values for each internal standard. For this we use the
+`matchValues()` function from the `r Biocpkg("MetaboAnnotation")` package using
+the `MzRtParam` method to identify all chromatographic peaks with similar *m/z*
+(+/- 50ppm) and retention time (+/- 10 seconds) to the internal standard's
+values. We perform this matching separate for each sample and select for each
+internal standard, in each sample, using the `filterMatches()` and
+`SingleMatchParam`, the chromatographic peak with the highest intensity.
 
-Once the different chromatographic peaks have been aligned, we can proceed to the
-next step in the data pre-processing pipeline, correspondence.
+```{r}
```

Comment:
Here I also modified the R code trying to make it a bit cleaner and splitting it into different code blocks.